packages/next/src/server/dev/hot-reloader-types.ts

```diff
@@ -0,0 +1,44 @@
+import type { IncomingMessage, ServerResponse } from 'http'
+import type { UrlObject } from 'url'
+import type { Duplex } from 'stream'
+import type { webpack } from 'next/dist/compiled/webpack/webpack'
+import type getBaseWebpackConfig from '../../build/webpack-config'
+import type { RouteMatch } from '../future/route-matches/route-match'
+
+export interface NextJsHotReloaderInterface {
+  activeConfigs?: Array<Awaited<ReturnType<typeof getBaseWebpackConfig>>>
+  serverStats: webpack.Stats | null
```

Comment:
webpack.Stats is probably hard to implemented for non webpack implementations

Reply:
This is only used in `logErrorWithOriginalStack`, we'll want to replace the usage of `serverStats` for this.